app/src/docs/components/ApiPage/index.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import React from 'react'
+
+import DocsLayout from '../DocsLayout'
+// $FlowFixMe
+import ApiPageContent from './Content.mdx'
+
+const subNav = {
+    clientLibraries: 'Client Libraries',
+    authentication: 'Authentication',
```

Comment:
Maybe these could be read automatically by traversing `ScrollableAnchor` elements in the child page.

Reply:
I'm not sure how to do this in a straight forward way so lets leave it for now and come back to it in a future ticket.